pkg/hostpath/constants.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
This code copied verbatim? If so, from where? Do we in fact need it?

Reply:
yeah actually I created the file itself and then moved some of the constants / types shared between hostpath.go and nodeserver.go (and their test.go files) into it

I'll remove the k8s copyright